custom/scripts/eks/README.md

```diff
@@ -0,0 +1,119 @@
+# Setup EKS And Configure Kafka
+A terraform script to create a managed kubernetes cluster on AWS EKS with the spot Instance.This script also gives you an option to setup kafka along with prometheus and kafka-provisioned grafana and also litmus portal.
```

Comment:
A terraform script to create a managed Kubernetes cluster on AWS EKS with the Spot Instance. This script also gives you an option to set up Kafka along with Prometheus and Kafka-provisioned Grafana and also a litmus portal.